doc/lua_api.txt

```diff
@@ -3305,6 +3306,20 @@ Definition tables
         action = func(pos, node, active_object_count, active_object_count_wider),
     }
 
+### LBM (LoadingBlockModifier) definition (`register_lbm`)
+
+    {
+        name = "modname:replace_legacy_door",
+    --  In the following field, also non-registered nodes will work.
+    --  groups (as of group:groupname) will work as well.
+        nodenames = {"default:lava_source"},
+        run_at_every_load = false,
+    --  ^ Whether to run the LBM's action every time a block gets loaded,
+    --    and not just for blocks that were saved last time before LBMs were
+    --    introduced to the world.
```

Comment:
The comments here are inconsistent: `nodenames` is documented before its example and `run_at_every_load` after.